src/main/java/org/tron/common/runtime/vm/program/Program.java

```diff
@@ -1432,6 +1415,7 @@ public StaticCallModificationException() {
   }
 
   public static class Exception {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this class "final" or add a public constructor. ([squid:S2974](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2974))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1681&issues=AWbnEHPQSgN4uCcljsr0&open=AWbnEHPQSgN4uCcljsr0)

[ISSUE_KEY]: AWbnEHPQSgN4uCcljsr0